src/PKSim.Core/Services/SimulationPKParametersImportTask.cs

```diff
@@ -14,42 +16,26 @@ public interface ISimulationPKParametersImportTask
 
    public class SimulationPKParametersImportTask : ISimulationPKParametersImportTask
    {
-      private readonly ISimulationPKAnalysesImporter _pkAnalysesImporter;
-      private readonly IEntitiesInContainerRetriever _quantityRetriever;
+      private readonly ICoreSimulationPKParametersImportTask _coreSimulationPKParametersImportTask;
 
-      public SimulationPKParametersImportTask(ISimulationPKAnalysesImporter pkAnalysesImporter, IEntitiesInContainerRetriever quantityRetriever)
+      public SimulationPKParametersImportTask(ICoreSimulationPKParametersImportTask coreSimulationPKParametersImportTask)
       {
-         _pkAnalysesImporter = pkAnalysesImporter;
-         _quantityRetriever = quantityRetriever;
+         _coreSimulationPKParametersImportTask = coreSimulationPKParametersImportTask;
       }
 
       public async Task<SimulationPKParametersImport> ImportPKParameters(PopulationSimulation populationSimulation, string fileFullPath, CancellationToken cancellationToken)
       {
-         var importedPKAnalysis = await importPKAnalysesFromFile(fileFullPath, cancellationToken);
+         var importedPKAnalysis = await _coreSimulationPKParametersImportTask.ImportPKParameters(fileFullPath, populationSimulation, cancellationToken);
+         cancellationToken.ThrowIfCancellationRequested();
          validateConsistencyWithSimulation(populationSimulation, importedPKAnalysis);
-         addImportedPKToLogForSuccessfulImport(importedPKAnalysis);
          return importedPKAnalysis;
       }
 
-      private void addImportedPKToLogForSuccessfulImport(SimulationPKParametersImport pkParameterImport)
```

Comment:
This was also put to core and was consolidated when updated to latest core version. Unrelated to the changes here